src/drivers/differential_pressure/ms4525/ms4525_airspeed.cpp

```diff
@@ -496,12 +531,16 @@ ms4525_airspeed_main(int argc, char *argv[])
 	int ch;
 	const char *myoptarg = nullptr;
 
-	while ((ch = px4_getopt(argc, argv, "b:", &myoptind, &myoptarg)) != EOF) {
+	while ((ch = px4_getopt(argc, argv, "ab:", &myoptind, &myoptarg)) != EOF) {
 		switch (ch) {
 		case 'b':
 			i2c_bus = atoi(myoptarg);
 			break;
 
+		case 'a':
+			i2c_bus = PX4_I2C_ALL;
```

Comment:
Instead of having a magic fake I2C bus what about a separate call for starting all or a default argument?

Reply:
@dagar I'm not sure I understood. For example I could set a flag in case 'a' and then if the command is 'start' I call meas_airspeed::start if I want to try all busses or I call meas_airspeed::start_bus if I want to start a specific bus?
```
if (!strcmp(argv[myoptind], "start")) {
		if(start_all){
			return meas_airspeed::start(i2c_bus);
		} else {
			return meas_airspeed::start_bus(i2c_bus);
		}
}
```
And in the regular ::start I always try all busses instead of checking for the magic bus.